src/main/js/controls/Graph/Graph.js

```diff
@@ -377,6 +381,82 @@ class Graph extends Construct {
         return this;
     }
 
+    /**
+     * Updates the graph axisData and content.
+     *
+     * @returns {Graph} - Graph instance
+     * @param {Array} graphData - Input array that holds updated values and key
+     */
+    reflow(graphData) {
+        this.config.axis.x.domain = getDomain(
+            this.config.axis.x.type,
+            this.config.axis.x.lowerLimit,
+            this.config.axis.x.upperLimit
+        );
+        const width = getXAxisWidth(this.config);
+        scaleGraph(this.scale, this.config);
+        const axisData = d3.svg
+            .axis()
+            .scale(this.scale.x)
+            .ticks(
+                Math.max(
+                    Math.ceil(width / constants.MAX_TICK_VARIANCE),
+                    constants.MIN_TICKS
+                )
+            )
+            .orient(this.config.axis.x.orientation);
+
+        const svg = d3
+            .selectAll("svg")
+            .selectAll(`.${styles.axis}.${styles.axisX}`)
+            .data(this.config.axis.x.domain);
+        svg.enter();
+        svg.transition()
+            .attr("class", styles.axis)
+            .attr("class", styles.axisX)
+            .attr("aria-hidden", !this.config.axis.x.show)
+            .attr(
+                "transform",
+                `translate(${getXAxisXPosition(
+                    this.config
+                )}, ${getXAxisYPosition(this.config)})`
+            )
+            .call(axisData);
+        svg.exit().remove();
+
+        if (graphData && this.contentKeys.includes(graphData.key)) {
```

Comment:
`includes` has no IE [support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes), make sure you are staying consistent with what browsers our library supports (IE10+)

Reply:
So on the MDN documentation `includes`, `lastIndexOf` and `indexOf` are not supported on IE. But during my testing  on IE 10, they all were working as expected.

Reply:
@pranav300 can you provide more information on how this was tested in IE10?  It is hard to believe that it just "works".  Using the IE console, `Array.includes()` returns undefined in both IE 11 and in IE 11 with compatibility mode set to IE 10 but does work in IE Edge

IE 11
![image](https://user-images.githubusercontent.com/44301749/82577587-cd2a1e80-9b50-11ea-8860-80479fc491c6.png)

IE 11 in IE 10 compatibility mode
![image](https://user-images.githubusercontent.com/44301749/82577636-e03cee80-9b50-11ea-9cd4-9dbeb746e88d.png)

IE Edge
![image](https://user-images.githubusercontent.com/44301749/82577737-02367100-9b51-11ea-8997-d9729a002980.png)